models/cnn/cnn.py

```diff
@@ -0,0 +1,167 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class CNN(nn.Module):
+    
```

Comment:
Please add a comment/docstring explaining from where this architecture stems (ex. paper reference) and quickly explain the structure.

Also explain what the different parameters do and add a demo config that one can put into the json file (if your model supports configuration as so)

Reply:
I will add the config options in my PR, no worries. The pre-train flag can be removed I think but I will check and leave it as necessary